.github/scripts/parse_rpt.py

```diff
@@ -1,33 +1,62 @@
 import sys
```

Comment:
Can you please write a brief summary of how this script just for the documentation purposes? it would be useful if someone wants to add more functionalities and doesn't want to break this script.